doc/manual.md

```diff
@@ -4973,6 +4973,15 @@ There is also a way which can be used to forbid certain effects:
 
 The `forbids` pragma defines a list if illegal effects - proc types not using that effect are all valid use cases.
 
+In the example below, the `ProcType1` procedure type is a subtype of `ProcType1`:
+
+.. code-block:: nim
+  type MyEffect = object
+  type ProcType1 = proc (i: int) {.forbids: [MyEffect].}
+  type ProcType2 = proc (i: int)
+
+Calling `ProcType2` in place of `ProcType1` is valid because `ProcType2` doesn't support any tag, and thus it doesn't allow `MyEffect` either.
+
```

Comment:
Well no, ProcType2 actually allows *any* effect, it's unspecified aka "don't care".

Reply:
I've changed/corrected the subtyping: https://github.com/nim-lang/Nim/blob/3fc7865c1b706a71cb13662d7e72d32599c61ca8/tests/effects/teffects19.nim
I'm using the same test in the manual and I've tried to explain the subtyping relation.